meta-oasis-vm/recipes-containers/container-host-config/container-host-config/registries.conf

```diff
@@ -0,0 +1,25 @@
+# This is a system-wide configuration file used to
+# keep track of registries for various container backends.
+# It adheres to TOML format and does not support recursive
+# lists of registries.
+
+# The default location for this configuration file is /etc/containers/registries.conf.
+
+# The only valid categories are: 'registries.search', 'registries.insecure', 
+# and 'registries.block'.
+
+[registries.search]
+registries = ['docker.io', 'registry.fedoraproject.org', 'quay.io', 'registry.access.redhat.com']
```

Comment:
Should we use the new format, as this one appears to be deprecated: https://github.com/containers/image/blob/main/docs/containers-registries.conf.5.md#version-1-format---deprecated 

Also, would it be safer to keep this empty, so that fully qualified image names need to always be used?

Reply:
Good idea.